commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java

```diff
@@ -63,7 +64,7 @@ public void openPlatform(TestInfo testInfo) {
   }
 
   @AfterEach
-  public void afterEach() {
```

Comment:
Not sure we should rename it to "signOut". This method may contain more things in the future, not related to logout. If you want something more meaningful you can create a signOut method instead and call it instead afterEach method IMO.

Reply:
I rename this method to override the signOut() method in LogoutTestIT.java which fails because of 2 consecutive signOut

Reply:
So we must find another way, probably by adapting the LogoutTestIT class instead of doing this change in Base class. With this change, the method is nowcalled signOut whereas it does not do only sign out, it also deals with Selenium screenshots, so that does not make sense.

Reply:
we can create an other method with name signOut(), so finally we will have 2 @AfterEach in Base class (one for public void afterEach and the other for public void signOut()). WDYT ?

Reply:
Can't you test if the user is not already logout before doing it ?

Reply:
ah yes we can do this

Reply:
done